pupil_src/launchables/world.py

```diff
@@ -305,9 +305,17 @@ def handle_notifications(n):
                                         'actor': p.class_name,
                                         'doc': p.on_notify.__doc__})
 
+        width, height = 1280, 720
+        width += icon_bar_width
+        default = width, height
+
+        width, height = session_settings.get('window_size', default)
```

Comment:
I dont like this kind of fix. Can we not rather catch this when the settings are written to file?

Reply:
Agreed! Will change tomorrow.